java/ql/src/semmle/code/java/frameworks/spring/SpringErrors.qll

```diff
@@ -0,0 +1,22 @@
+import java
+private import semmle.code.java.dataflow.ExternalFlow
+
+private class SpringErrorModel extends SummaryModelCsv {
+  override predicate row(string row) {
+    row =
+      [
+        "org.springframework.validation;Errors;true;addAllErrors;;;Argument[0];Argument[-1];taint",
+        "org.springframework.validation;Errors;true;getAllErrors;;;Argument[-1];ReturnValue;taint",
+        "org.springframework.validation;Errors;true;getFieldError;;;Argument[-1];ReturnValue;taint",
+        "org.springframework.validation;Errors;true;getFieldErrors;;;Argument[-1];ReturnValue;taint",
+        // for review: excluding because I think field values are unlikely to carry meaningful taint, though I may be wrong.
```

Comment:
It's not clear just from the interface where we'd propagate taint from (`this`, or some associated object chosen by the implementation?), so I'm inclined to agree